modules/minecraft-servers.nix

```
@@ -567,6 +586,10 @@ in
                 example = options.services.minecraft-servers.managementSystem.example;
               };
             };
+
+            config = mkIf config.enable {
+              allowedSymlinks = [ "/nix/store" ];
+            };
```

```suggestion

            config = {
              allowedSymlinks = [ "/nix/store" ];
            };
```
(And remove the `config` reference at the top of the section)

I don't think this being set will have any side effects if the server isn't enabled to begin with, so having it here makes it kinda redundant. Plus, it would prevent seeing the value of `allowedSymlinks` if the server is disabled. (Though that doesn't really matter in practice.)

If there is a notable reason why this needs to be gated under `enable` though, then it can stay.